packages/cli-doctor/src/tools/versionRanges.ts

```diff
@@ -3,7 +3,7 @@ export default {
   NODE_JS: '>= 16',
   YARN: '>= 1.10.x',
   NPM: '>= 4.x',
-  RUBY: '>= 2.7.6',
+  RUBY: '>= 2.7.10',
```

Comment:
```suggestion
  RUBY: '>= 2.6.10',
```

This should align with: https://github.com/facebook/react-native/blob/main/template/Gemfile

cc @cipolleschi